ui/src/app/store/bootresource/slice.ts

```diff
@@ -46,7 +46,9 @@ const bootResourceSlice = createSlice({
           model: BootResourceMeta.MODEL,
           method: "delete_image",
         },
-        payload: params,
+        payload: {
+          params,
+        },
```

Comment:
accidentally set all of these wrong, lolz

Reply:
I've done that before.